### PR TITLE
Add help comments in Makefile

### PR DESCRIPTION
**Pull request by hsorellana**
**What this PR does / Why we need it**:
Adds description to make command and displays it when running make help

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -739,6 +746,10 @@ define fast_test_folder
 	$(foreach dir, $(wildcard $(1)/*/.), $(call fast_test_folder, $(dir)))
 endef
 
+## ####################################
```
Comment (aLekSer) [inline Makefile:749]: This should be before `define test_folder`

**Review by aLekSer — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (aLekSer) [reply Makefile:749]: Even before `build:`

File: Makefile [hunk 2]
```diff
@@ -699,6 +699,13 @@ api/%.swagger.json: api/%.proto third_party/ build/toolchain/bin/protoc$(EXE_EXT
 		-I $(REPOSITORY_ROOT) -I $(PROTOC_INCLUDES) \
 		--swagger_out=logtostderr=true,allow_delete_body=true:$(REPOSITORY_ROOT)
 
+## ####################################
+## # Utilities
+##
```
Comment (aLekSer) [inline Makefile:704]: No need for this since it already belongs to:
``` 
## # Protobuf
```
section
```suggestion
```

File: Makefile (hunk 1 above)
Comment (hsorellana) [reply Makefile:749]: ohh you're right!

**Review by aLekSer — APPROVED** [review]
Thanks, now it is better.

**Pull request merged**